lib/modules/datasource/github-runners/index.ts

```diff
@@ -23,6 +23,10 @@ export class GithubRunnersDatasource extends Datasource {
       { version: '18.04', isDeprecated: true },
       { version: '16.04', isDeprecated: true },
     ],
+    ubuntu_arm64: [
+      { version: '24.04', isStable: false },
+      { version: '22.04', isStable: false },
```

Comment:
Are you marking them as unstable because GitHub has them in "public preview"? I would lean towards treating them both as stable seeing as ubuntu is stable

Reply:
That's correct, though I don't have a strong opinion so I'm happy to mark them as stable 🙂

Reply:
226d36b73